modules/epang/place/main.nf

```diff
@@ -0,0 +1,41 @@
+process EPANG_PLACE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::epa-ng=0.3.8" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/epa-ng:0.3.8--h9a82719_1':
+        'quay.io/biocontainers/epa-ng:0.3.8--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(queryaln), path(referencealn), path(referencetree)
+
+    output:
+    tuple val(meta), path("*.epa_result.jplace"), emit: jplace
+    path "*.epa_info.log"                       , emit: log
+    path "versions.yml"                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    epa-ng \\
+        $args \\
+        --threads $task.cpus \\
+        --query $queryaln \\
+        --ref-msa $referencealn \\
+        --tree $referencetree \\
+        --model $meta.model
```

Comment:
Is the model sample specific? If yes, it might be fine to have it in the meta map (or as part of the above input tuple), if no, it might be better added as `val` (or `path` if file, but it seems to me its a string?) under `input`.

Reply:
It's actually tied to the reference, so there's a possibility that it's not, but I kept it in `meta` anyway for the moment as I was suggested not to have other value parameters in another module.